web/common

```diff
@@ -0,0 +1 @@
+../node/common
```

Comment:
Should something common be placed at the top directory rather than at either `node/` or `web/` if `../node/common` is used for both node and web? Or, is it in the node directory since it is still "node" style or something like that?

Reply:
> It's a good idea. Let me move common from the node folder.

@supl Thank you! Is it done by another PR?

Reply:
@jnmt 
Yes. I will do that in another PR.
(It's not done yet sorry)

Reply:
@supl OK, thanks! It's not an urgent matter.